### DISCUSSION TITLE
Rework keyboard shortcuts setting

### DISCUSSION THREAD

**Discussion by vietbunasd** [goastian/midori-desktop#40] [Ideas]
It's very difficult to determine whether a key combination has already been used for a shortcut or not, and when I make a mistake, I have to restore everything to default, which is inconvenient.
![image](https://github.com/goastian/midori-desktop/assets/44780347/8d81cc13-5030-47e7-8509-b96ba9b74e82)
Therefore, I propose redesigning the keyboard shortcut settings, emulating the Vivaldi browser, to make the settings more intuitive and user-friendly. 

1. This includes alerting the user when a key combination is assigned to a different shortcut, asking if they would like to modify it,
2. Show which key combination assigned to which action.
3. Add a restore button to an individual shortcuts to restore only shortcut I need to default instead of restore everything to default. 
4. Additionally, eliminates the need to restart the browser in order for the shortcut to take effect.

![image](https://github.com/goastian/midori-desktop/assets/44780347/de59ea5d-2d62-4e88-8743-151d39414565)

**Comment by Ponchale** [comment]
Fix problem
